README.md

```diff
@@ -0,0 +1,127 @@
+   git clone git@github.com:Movgto/hello-rails-back-end.git
+    ```
+2. Navigate to the project folder:
+   ```bash
+   cd hello-rails-back-end
+   ```
+### Prerequisites <a name="prerequisites"></a>
+Before you can run this app, make sure you have the following prerequisites and configurations in place:
+
+- **React front-end:** <a name="frontend-instructions"></a> Remember that this application is only the API for a React front-end, so you must also follow the instructions to setup the front-end to have both working together. See the instructions [here](https://github.com/Movgto/hello-react-front-end).
+
+- **Ruby**: Ensure that Ruby is installed on your machine. You can check if it's installed by running `ruby -v` in your terminal. If it's not installed, you can download and install it from the [official Ruby website](https://www.ruby-lang.org/en/documentation/installation/).
+
+- **Ruby on Rails**: You'll need the Ruby on Rails framework installed. If you don't have it yet, you can install it using the following command:
+  ```
+  gem install rails
+  ```
+
+- **Code Editor**: You should have a code editor like Visual Studio Code (VSCode) installed. You can download VSCode from the [official website](https://code.visualstudio.com/).
+
+- **Git**: Git is essential for version control. Make sure you have Git installed and configured on your system. You can download Git from the [official Git website](https://git-scm.com/downloads).
+
+- **PostgreSQL**: Ensure that PostgreSQL is installed and properly configured on the server where you are developing the project. This project assumes the use of PostgreSQL as the default database. You can download PostgreSQL from the [official PostgreSQL website](https://www.postgresql.org/download/).
+
+- **Ruby on Rails Configuration**: Make sure your Ruby on Rails project is configured to use PostgreSQL as the default database.
+
+### 📖 Usage <a name="usage"></a>
+Follow these steps to use the app:
+
+### Backend with Ruby on Rails:
+
+Open your terminal.
+
+Navigate to the directory where the project files are located:
+
+Copy code
+```bash
+cd /path/to/your/project/files
+```
+Start the Rails server:
+
+Copy code
+```bash
+rails server
+```
+
+**Congratulations, now you have both applications running!**
+
+Now just open your browser and go to [this link](localhost:3000) and see them in action. Remember to also follow these [instructions](#frontend-instructions) for running the front-end application with React.
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Gumaro Monroy Vazquez**
+- GitHub: [GitHub](https://github.com/Movgto)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/gumaro-monroy-vazquez-1705aa165/)
+- WellFound: [Wellfound](https://wellfound.com/u/maro-monroy)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/Movgto/react-rails-greetings/issues) and contribute to the project.
```

Comment:
- [ ] Kindly ensure that the issue link is working, I got 404 error in the process of accessing it. 🙏